### PR TITLE
Strokeborder color not shown in inspect when applying it as a style

### LINKED ISSUE #1855: Stroke/border color not shown in inspect, when applying it as a Style.
**Describe the bug**
When you apply a border color as a Style (via Figma, so not as a token), the style won't show up in the Inspect panel.
This is working for other styles:
- typography
- color as a fill
- shadow

Only not working for:
- Stroke/border style/color

**To Reproduce**
See screencapture.

**Expected behavior**
Show the border color in the inspect panel as a Style, just like typography and color as a fill.

**Screenshots or Screencasts**
https://user-images.githubusercontent.com/109062656/236131008-c580e1e1-3fa4-496b-90f9-fa9582c5a467.mov

### PR DESCRIPTION
**Pull request by robinhoodie0823**
### Why does this PR exist?
The current plugin has the problem of not showing stroke styles applied on nodes in inspect view.

Closes #1855  

### What does this pull request do?
The PR resolves the problem and show the applied styles on node in inspect view.

### Testing this change
Updated `getAppliedStylesFromNode` function to get applied `strokes` styles from node. (Added a condition to check if the node has `strokes` property).

### Additional Notes (if any)

https://github.com/tokens-studio/figma-plugin/assets/103296157/89bf1cd1-acdc-494a-af42-83d623ae5bca

### CODE REVIEW

File: .changeset/wicked-lobsters-fetch.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@tokens-studio/figma-plugin': patch
+---
+
+Allow to inspect stroke styles on node
```
Comment (six7) [inline .changeset/wicked-lobsters-fetch.md:5]:
```suggestion
Fixed an issue where stroke styles where not shown in Inspect
```

File: src/plugin/getAppliedStylesFromNode.ts [hunk 2]
```diff
@@ -89,6 +89,41 @@ export default function getAppliedStylesFromNode(node: BaseNode): SelectionStyle
     }
   }
 
+  if ('strokes' in node) {
```
Comment (six7) [inline src/plugin/getAppliedStylesFromNode.ts:92]: this feels very repetitive of the if statement on line 58. let's try to extract this into a reusable function:

```
function getStyleFromNode(node: BaseNode, styleIdBackupKey: string, styleType: 'fills' | 'strokes'): SelectionStyle[] {
  const localStyles: SelectionStyle[] = [];

  const localStyle = getLocalStyle(node, styleIdBackupKey, styleType);
  if (localStyle) {
    const paint = localStyle.paints[0];
    let styleObject: SingleColorToken | null = {} as SingleColorToken;
    if (paint.type === 'SOLID') {
      const { r, g, b } = paint.color;
      const a = paint.opacity;
      styleObject.value = figmaRGBToHex({ r, g, b, a });
    } else if (paint.type === 'GRADIENT_LINEAR') {
      styleObject.value = convertFigmaGradientToString(paint);
    } else {
      styleObject = null;
    }
    const normalizedName = localStyle.name.split('/').map(section => section.trim()).join('.');
    if (styleObject) {
      const styleTypeConstant = styleType === 'fills' ? Properties.fill : Properties.borderColor;
      localStyles.push({...styleObject, name: normalizedName, type: styleTypeConstant});
    }
  }

  return localStyles;
}
```

and 

```
...

if ('strokes' in node) {
  const styleIdBackupKey = 'strokeStyleId_original';
  localStyles.push(...getStyleFromNode(node, styleIdBackupKey, 'strokes'));
}

if ('fills' in node) {
  const styleIdBackupKey = 'fillStyleId_original';
  localStyles.push(...getStyleFromNode(node, styleIdBackupKey, 'fills'));
}

...
```

Comment (six7) [reply src/plugin/getAppliedStylesFromNode.ts:92]: (note: this was generated using Copilot, there might be mistakes in the code)

**Review by six7 — CHANGES_REQUESTED** [review]
some optimizations around reusability of code

**Review by six7 — APPROVED** [review]
Approved from a code side. @esthercheran can you review functionally?

**Review by SorsOps — APPROVED** [review]

**Review by esthercheran — COMMENTED** [review]
The functionality is working as expected

**Review by esthercheran — APPROVED** [review]

**Pull request merged**